tests/org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/highlighting/XtendHighlightingCalculatorTest.java

```diff
@@ -132,11 +140,13 @@ protected XtendClass clazz(String string) throws Exception {
 	}
 	
 	@Test public void testLiteral() {
+		strictMode = true;
 		expectInsignificant(0, 3);
 		expectInsignificant(3, 2);
+		expectTemplateLineBreak(5, 1);
 		expectInsignificant(6, 1);
+		expectTemplateLineBreak(14, 1); //TODO shouldn't this be POSSIBLE_LINE_BREAK
```

Comment:
POTENTIAL_LINE_BREAK is used for lines that contain only expressions (no semantic ws). IF all expressions evaluate to null, there is no line emitted. TEMPLATE_LINE_BREAK seems about right to me here and for the other TODOs too.